EDAC (Error Detection And Correction) reporting
EDAC is a subsystem along with hardware-specific drivers designed to
report hardware errors. These are low-level errors that are reported
in the CPU or supporting chipset or other subsystems:
memory errors, cache errors, PCI errors, thermal throttling, etc..
If unsure, select 'Y'.

The mailing list for the EDAC project is linux-edac@vger.kernel.org.
Depends: HAS_IOMEM && EDAC_SUPPORT && RAS
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:20:28.586610308 -0400
@@ -637,8 +637,6 @@
 CONFIG_DPTF_POWER=m
 CONFIG_DPTF_PCH_FIVR=m
 CONFIG_ACPI_WATCHDOG=y
-# CONFIG_ACPI_EXTLOG is not set
-CONFIG_ACPI_ADXL=y
 # CONFIG_ACPI_CONFIGFS is not set
 CONFIG_ACPI_PFRUT=m
 CONFIG_ACPI_PCC=y
@@ -7622,28 +7620,7 @@
 CONFIG_INFINIBAND_OPA_VNIC=m
 CONFIG_EDAC_ATOMIC_SCRUB=y
 CONFIG_EDAC_SUPPORT=y
-CONFIG_EDAC=y
-CONFIG_EDAC_LEGACY_SYSFS=y
-# CONFIG_EDAC_DEBUG is not set
-CONFIG_EDAC_DECODE_MCE=m
-CONFIG_EDAC_GHES=y
-CONFIG_EDAC_AMD64=m
-CONFIG_EDAC_E752X=m
-CONFIG_EDAC_I82975X=m
-CONFIG_EDAC_I3000=m
-CONFIG_EDAC_I3200=m
-CONFIG_EDAC_IE31200=m
-CONFIG_EDAC_X38=m
-CONFIG_EDAC_I5400=m
-CONFIG_EDAC_I7CORE=m
-CONFIG_EDAC_I5000=m
-CONFIG_EDAC_I5100=m
-CONFIG_EDAC_I7300=m
-CONFIG_EDAC_SBRIDGE=m
-CONFIG_EDAC_SKX=m
-CONFIG_EDAC_I10NM=m
-CONFIG_EDAC_PND2=m
-CONFIG_EDAC_IGEN6=m
+# CONFIG_EDAC is not set
 CONFIG_RTC_LIB=y
 CONFIG_RTC_MC146818_LIB=y
 CONFIG_RTC_CLASS=y
